src/aics-image-viewer/components/AxisClipSliders/index.tsx

```diff
@@ -172,18 +182,14 @@ export default class AxisClipSliders extends React.Component<AxisClipSlidersProp
   makeSliderUpdateFn(axis: string) {
     return (values: number[]) => {
       if (this.props.setAxisClip) {
-        const step = 1;
-        const stepEpsilon = 0.04;
+        // get a value from -0.5..0.5
         const max = this.props.numSlices[axis];
-        const isActiveAxis = this.getActiveAxis() === axis;
-        const thicknessReduce = isActiveAxis ? step - stepEpsilon : 0.0;
```

Comment:
I believe stepEpsilon was there because in 2d modes we had some difficulty showing very thin slabs (~1 slice) and needed to guarantee at least some minimum thickness. Have you seen any issues with single slice viewing in any of the 3 projections?  The old issue was an artifact of the ray marching algorithm in the shader having some tiny offsets to try to anti-alias the render.  We can discuss but if nothing looks wrong then that is great.

Reply:
Nothing was looking wrong to me when I tested it. If that's the case though it might be worth adding a minimum thickness value anyways, since slice thickness can get basically arbitrarily small for more and more detailed models.